tools/server/webui/src/assets/iframe_sandbox.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
```

Comment:
Normally to have better security, iframes usually be on a dedicated domain. I'm not sure what is the risk of having iframe on the same domain as main website, but probably add a note about security for more transparency to end-user

Note that the fact that we use iframe here is to avoid the executed code to mess up with main UI's state, but it doesn't mean we aims to protect the code from reading `localStorage` or to access other same-domain shared stuff. But it's fine to do this way rn as we don't have much choices.

Reply:
Since we are using `sandbox`, it sets the [unique origin](https://www.w3schools.com/tags/att_iframe_sandbox.asp).
> When the sandbox attribute is present, and it will:
> * treat the content as being from a unique origin